nodejs/week1/avg.js

```diff
@@ -0,0 +1,9 @@
+const myNumbers = process.argv.slice(2);
+let sum = 0;
+
+for (let i = 0; i < myNumbers.length; i++) {
+  sum += Number(myNumbers[i]);
+}
+
+let average = sum / myNumbers.length || 0;
```

Comment:
Great to check for zero length! Otherwise you would get a division by zero error. 👏